src/Microsoft.DotNet.Interactive.ApiCompatibility.Tests/ApiCompatibilityTests.Interactive_api_is_not_changed.approved.txt

```diff
@@ -229,10 +229,12 @@ Microsoft.DotNet.Interactive
     public System.Threading.Tasks.Task<Microsoft.DotNet.Interactive.Connection.ProxyKernel> ConnectProxyKernelOnDefaultConnectorAsync(System.String localName, System.Uri remoteKernelUri, System.String[] aliases = null)
     public System.Void Dispose()
   public class KernelInfo
-    .ctor(System.String localName, System.String[] aliases = null)
+    .ctor(System.String localName, System.String[] aliases = null, System.Boolean isProxy = False, System.Boolean isComposite = False)
     .ctor(System.String localName, System.String languageName, System.String languageVersion, System.String[] aliases)
     public System.String[] Aliases { get; set;}
     public System.String DisplayName { get; set;}
+    public System.Boolean IsComposite { get;}
+    public System.Boolean IsProxy { get;}
```

Comment:
These should be mutable like the others.

Reply:
ok